### PR TITLE
feat(attribute): Add `HasCx` trait and corresponding tests for managing SVG `cx` attribute.

### PR DESCRIPTION
**Pull request by terabytesoftw**
# Pull Request

| Q            | A                                                                  |
| ------------ | ------------------------------------------------------------------ |
| Is bugfix?   | ❌                                                              |
| New feature? | ✔️                                                              |
| Breaks BC?   | ❌                                                              |

## Summary by CodeRabbit

* **New Features**
  * Added SVG cx attribute support via an immutable, fluent API accepting float, int, string, or null.

* **Tests**
  * Added comprehensive tests covering rendering, value handling, immutability, and edge cases for the cx attribute.

* **Documentation**
  * CHANGELOG updated with an entry for the new cx attribute.

<sub>✏️ Tip: You can customize this high-level summary in your review settings.</sub>

### CODE REVIEW

File: src/Values/SvgProperty.php [hunk 1]
```diff
@@ -21,6 +21,15 @@
...
 enum SvgProperty: string
 {
+    /**
+     * `cx` - Center x-coordinate of a circle or ellipse element.
+     *
+     * Defines the horizontal position of the center point of the element in the current user coordinate system.
+     *
+     * @link https://developer.mozilla.org/en-US/docs/Web/SVG/Attribute/cx
+     */
+    case CX = 'cx';
```
[the remark this answers is not in the document]
Comment (terabytesoftw) [reply src/Values/SvgProperty.php]: fixed.

**Comment by terabytesoftw** [comment]
@coderabbitai review
